src/js/gutenberg-plugins/content-resizing-plugin.js

```diff
@@ -237,6 +241,15 @@ const ContentResizingPlugin = () => {
 			updateWith.length
 		);
 		resetStates();
+
+		// If no edited values in post trigger save.
+		if ( ! isDirty ) {
+			await dispatch( 'core' ).saveEditedEntityRecord(
```

Comment:
Does this trigger an autosave or a normal save? Or maybe in Gutenberg it's the same thing? I'm seeing the `saveEntityRecord` function has an `isAutosave` option but don't see the same in this function, so just want to ensure we're using the right thing here

Reply:
> Does this trigger an autosave or a normal save?

@dkotter this is a normal save. As Gutenberg provides information on whether the post is dirty or not, we can make sure that the post doesn't contain any other changes than our changes, we are good to trigger the normal save here. Please let me know if you think we should trigger autosave here instead of the normal save. 

> Or maybe in Gutenberg it's the same thing?

Normal save is different from autosave in Gutenberg as well, like the classic editor.